qt5-tools/mingw-w64-static/PKGBUILD

```diff
@@ -24,7 +24,7 @@
 _qt_module=qttools
 pkgname=mingw-w64-qt5-tools-static
 pkgver=5.15.1
-pkgrel=1
+pkgrel=2
```

Comment:
There is no need to update the package if already worked around the issue - so don't increase pkgrel.

Reply:
Ah, that makes sense, updated.